tests/test_test_utils.py

```diff
@@ -137,5 +137,5 @@ def test_iterable_eq_passes_two_numpy_array_quantities():
     """
     values = [1, 2, 3]
     a = numpy.array(values) * u.V
-    b = numpy.array(a) * u.V
```

Comment:
This was giving a warning about stripping values from `a` in the process of being cast to a `numpy.array`. While that's the intended end result, I think this tests just fine by duplicating the two lines.